composer.json

```diff
@@ -40,45 +40,44 @@
         "sonata-project/form-extensions": "^1.4",
         "sonata-project/twig-extensions": "^1.3",
         "symfony/config": "^4.4 || ^5.2",
-        "symfony/console": "^4.4",
+        "symfony/console": "^4.4 || ^5.2",
```

Comment:
Can we bump this to 5.3? 

Support for 5.2 will end next month: https://symfony.com/releases/5.2

Reply:
It would be nice to do that across al repositories of Sonata. I will let that for another moment.